burn-derive/src/config/analyzer_enum.rs

```diff
@@ -72,11 +72,11 @@ impl ConfigEnumAnalyzer {
     fn gen_serialize_fn(&self) -> TokenStream {
         let enum_name = self.serde_enum_ident();
         let variants = self.data.variants.iter().map(|variant| {
-            let variant_name = &variant.ident;
-            let (variant_input, variant_output) = self.gen_variant_field(variant);
+      let variant_name = &variant.ident;
+      let (variant_input, variant_output) = self.gen_variant_field(variant);
 
-            quote! { Self::#variant_name #variant_input => #enum_name::#variant_name #variant_output }
-        });
+      quote! { Self::#variant_name #variant_input => #enum_name::#variant_name #variant_output }
+    });
```

Comment:
That formatting seems odd to me and somehow fmt can't update it 🤔

Reply:
What would you like me to do to resolve this issue and merge the PR? `cargo fmt --all` results in a bit identical repository. Is the issue these two lines of whitespace? Should I modify them manually? You mention no changes should be made to `burn-derive`.  Would like to get this closed out, apologies on the silliness of these problems.

Reply:
It's due to the `quote!` macro. https://github.com/rust-lang/rustfmt/issues/8

If you comment out `quote!`, format, and uncomment, it'll do the right thing.

Reply:
aaaaaactshully it's due to the default `max_width = 100`. If you add a file called `rustfmt.toml` and in it is `max_width = 110` then format, it works. I'll open another PR about this since it causes changes elsewhere and deserves its own discussion.